arches_lingo/src/arches_lingo/components/scheme/report/SchemeLabel.vue

```diff
@@ -105,11 +115,57 @@ function editSectionValue(tileId: string) {
             :title-text="$gettext('Scheme Labels')"
             @open-editor="emits(OPEN_EDITOR)"
         >
-            <LabelViewer
-                :value="schemeInstance?.appellative_status"
-                @edit-label="editSectionValue"
-                @delete-label="deleteSectionValue"
-            ></LabelViewer>
+            <MetaStringViewer
+                :meta-strings="schemeInstance?.appellative_status"
+                :meta-string-text="metaStringLabel"
+                @edit-string="editSectionValue"
+                @delete-string="deleteSectionValue"
+            >
+                <template #name="{ rowData }">
+                    {{
+                        (rowData as AppellativeStatus)
+                            .appellative_status_ascribed_name_content
+                    }}
+                </template>
+                <template #type="{ rowData }">
+                    <ControlledListItem
+                        :value="
+                            (rowData as AppellativeStatus)
+                                .appellative_status_ascribed_relation
+                        "
+                    >
+                    </ControlledListItem>
+                </template>
+                <template #language="{ rowData }">
+                    <ControlledListItem
+                        :value="
+                            (rowData as AppellativeStatus)
+                                .appellative_status_ascribed_name_language
+                        "
+                    >
+                    </ControlledListItem>
+                </template>
+                <template #drawer="{ rowData }">
+                    <div>
+                        {{ $gettext("Bibliographic Sources") }}:
+                        <ResourceInstanceRelationships
+                            :value="
+                                (rowData as AppellativeStatus)
+                                    .appellative_status_data_assignment_object_used
+                            "
+                        ></ResourceInstanceRelationships>
+                    </div>
+                    <div>
+                        {{ $gettext("Contributors") }}:
```

Comment:
similar here with "naked" strings in HTML

Reply:
fixed, included colon in gettext.  🏓